flatten/flatten.py

```diff
@@ -0,0 +1,28 @@
+import unittest
```

Comment:
According to https://www.python.org/dev/peps/pep-0008/#imports combined with https://www.python.org/dev/peps/pep-0008/#blank-lines between imports and function definitions we should leave two blank lines.